packages/devtools/web/index.html

```diff
@@ -8,6 +8,154 @@
 
 <html>
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
We  might want to add an `gtags=enabled|disabled` query parameter to let users opt in-out.
We could launch first only enabling gtags when `gtags=enabled` allowing the feature to be off by default. There is a case a little lower down in this file where we toggle the dark theme based on a queryParam.

Reply:
I've removed the ga-confirm (I was using for testing).  Added &gtags=enabled | disabled | reset

Reply:
Done.